ha-singleton-service/README.md

```diff
@@ -3,44 +3,47 @@
 
 Everything needed to build this project is ${build.requirements}. See [Configure Maven for ${product.name} ${product.version}](https://github.com/jboss-developer/jboss-developer-shared-resources/blob/master/guides/CONFIGURE_MAVEN_JBOSS_EAP7.md#configure-maven-to-build-and-deploy-the-quickstarts) to make sure the environment is configured correctly for testing the quickstarts.
 
 
-## Use of ${jboss.home.name}
+## Use of ${jboss.home.name}-1 and ${jboss.home.name}-2
+
+This quickstart requires that you clone your `${jboss.home.name}` installation directory and run two servers. The installation path is described in detail here: [Use of ${jboss.home.name} and JBOSS_HOME Variables](https://github.com/jboss-developer/jboss-developer-shared-resources/blob/master/guides/USE_OF_${jboss.home.name}.md#use-of-eap_home-and-jboss_home-variables).
 
-In the following instructions, replace `${jboss.home.name}` with the actual path to your ${product.name} installation. The installation path is described in detail here: [Use of ${jboss.home.name} and JBOSS_HOME Variables](https://github.com/jboss-developer/jboss-developer-shared-resources/blob/master/guides/USE_OF_${jboss.home.name}.md#use-of-eap_home-and-jboss_home-variables).
+In the following instructions, replace `${jboss.home.name}-1` with the path to your first ${product.name} server and replace `${jboss.home.name}-2` with the path to your second cloned ${product.name} server.
 
+## Clone the ${product.name} Directory
 
-## Setting Up the Test Environment
+While you can run this example starting only one instance of the server, if you want to see the singleton behavior, you must start at least two instances of the server. Copy of the entire ${product.name} directory to a new location to use for the second cluster member.
 
-To demonstrate the singleton behavior, at least two application server instances must be started.
-Begin by making a copy of the entire ${product.name} directory to be used as second cluster member.
-Note that the example can be run on a single node as well, but without observation of the singleton properties.
+## Start the Servers with the HA Profile
 
-Start the two ${product.name} servers with the same HA profile using the following commands.
-Note that a socket binding port offset and a unique node name must be passed to the second server if the servers are binding to the same host.
-These logical node names are used in the log to identify which node is elected. 
+_Note: You must start the server using the HA profile or the singleton service will not start correctly._
+
+Start the two ${product.name} servers with the HA profile, passing a unique node ID. These logical node names are used in the log to identify which node is elected. If you are running the servers on the same host, you must also pass a socket binding port offset on the command line to start the second server.
+To start the servers, type the following commands.
```

Comment:
"Type" is the standard verbiage in all of our quickstart README files and all of our JBoss EAP documentation. We tell them to type the commands, but they know they can copy and paste.